accounts/keystore/keystore_plain_test.go

```diff
@@ -174,7 +174,7 @@ func TestV1_1(t *testing.T) {
 
 func TestV1_2(t *testing.T) {
 	t.Parallel()
-	ks := &keyStorePassphrase{"testdata/v1", LightScryptN, LightScryptP}
+	ks := &keyStorePassphrase{"testdata/v1", LightScryptN, LightScryptP, true}
```

Comment:
If setting this value to `false`, does the test fail?

Reply:
I think `skipKeyFileVerification` doesn't affect to the test result. It is only used in `StoreKey` function to determine whether decryption is needed or not!

Reply:
Can we simulate the failing cases so that we can make an unit test for this?

Reply:
Um.. We can't simulate the failing cases because we don't know why fail occurs when creating keystore file. By the way, there seems to be no test case about GetKey with wrong keystore file. It would be nice to add it, so I will proceed in another PR since this has nothing to do with the current PR.